src/config/index.ts

```diff
@@ -0,0 +1,47 @@
+export const ENV = process.env.NODE_ENV || 'development'
+
+function getEnv(obj: any, attr: string) {
+	return process.env[attr] || ''
+}
+
+function set<T>(defaultValue: T, envValues: Record<string, T> = {}): T {
+	const val = envValues[ENV]
+	if (val === undefined) {
+		return defaultValue
+	}
+	return val
+}
+
+const env: Record<string, string> = new Proxy({}, {get: getEnv})
+
+export const LOGGING = set(
+	{
+		appenders: {
+			out: {type: 'console'},
+		},
+		categories: {
+			default: {appenders: ['out'], level: 'info'},
+		},
+	},
+	{
+		development: {
+			appenders: {
+				out: {type: 'console'},
+			},
+			categories: {
+				default: {appenders: ['out'], level: 'debug'},
+			},
+		},
+	}
+)
+
+export const AUTHENTICATION = set({
```

Comment:
We would want Configuration classes per package/component rather than application global strings, constants. They wouldn't dirty up the global scope and they are easier to test, update,  share, maintain.

```
class AuthConfig {
String clientId
String clientSecret
.....
}
config = new AuthConfig(ENV.clientId || 'hello'...)
app = new App(config)
...
```